### ISSUE TITLE
RFE: Add a way to run service in portable environment

### ISSUE THREAD

**Issue by Prototik** [systemd/systemd#14882]
I have a use-case when I need to deploy some daemon (basically systemd service) and client for that daemon, and I want to do those with a portable service to simplify delivering binaries to the end user (no need to separately install daemon as portable and client as external executable).

It would be nice if some way exist to run some commands in environment of portable service (root image basically) to invoke my client from outside of portable itself (so grant user ability to use client and communicate with a daemon), but unfortunately I cannot find any appropriate solution for this.

I'm thinking about new flag in `systemd-run`, something like these:
```
contents of portable image:
/bin/daemon
/bin/client
/lib/systemd/system/daemon.service
...
```
```
portablectl attach image.raw # installs daemon.service
systemd-run --in-scope daemon.service /bin/client --args # spawn transient service with RootImage specified
```

`--in-scope` probably bad name for this (as it can be associated wrongly with systemd's scope), so better name required.

Tried some quirk:
```
$ systemd-run --property=RootImage=/path/to/image.raw /bin/client
Failed to find executable /bin/client: No such file or directory
```
Doesn't seems to work.

**Comment by poettering** [comment]
what precisely do you mean by "in environment of portable service"? just RootImage=? 

The systemd-run thing should just work, but there might be a bug there, as systemd-run might validate the path relative to the host...

which systemd version is this?

**Comment by Prototik** [comment]
Yeah, basically `RootImage=` is sufficent. systemd-run doesn't seems to be working with that way:
```
$ systemd-run --property=RootImage=/path/to/image.raw /bin/client
Failed to find executable /bin/client: No such file or directory
```

**Comment by poettering** [comment]
Duplicate of #13338, and fixed by 8038b99d0d5be4ee5cfc5d99c3cf62f26223fb7a. Closing
